jtesta/ssh-audit
# print config v2 Issue #191

@jtesta @daniejstriata
 - printconfig script
 - test_printconfig for tox testing
 - update globals for GUIDES_UPDATED date value
 - update ssh_audit for print_config argument and checks

Quotes should no longer be needed.
usage : ssh-audit.py --print-config Ubuntu 2204 Server
